tests/test_physical_systems/test_electric_motors.py

```diff
@@ -0,0 +1,253 @@
+import pytest
+import gym_electric_motor as gem
+from gym_electric_motor.physical_systems.electric_motors import (
+    ElectricMotor,
+    DcMotor,
+    DcExternallyExcitedMotor,
+    DcPermanentlyExcitedMotor
+)
+from gymnasium.spaces import Box
+import numpy as np
+
+#parameters for dc motor
+test_dcmotor_parameter = {"r_a": 16e-2, "r_e": 16e-1, "l_a": 19e-5, "l_e_prime": 1.7e-2, "l_e": 5.4e-1, "j_rotor": 0.025}
+test_dcmotor_nominal_values = dict(omega=350, torque=16.5, i=100, i_a=96, i_e=87, u=65, u_a=65, u_e=61)
+test_dcmotor_limits = dict(omega=350, torque=38.0, i=210, i_a=210, i_e=215, u=60, u_a=60, u_e=65)
```

Comment:
I'm not sure if it's good to set the limits lower then the nominal values for the voltages. (parameters starting with an u)